### PR TITLE
Typo "Teams Room device"→"Teams Rooms device"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/rooms/rooms-operations 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/rooms/rooms-operations.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/rooms/rooms-operations.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ The console is now back in its normal operation mode. The following procedure re
    1. Select **Sign Off**.
 5. Navigate to `C:\Users\Skype\AppData\Local\Packages\Microsoft.SkypeRoomSystem_8wekyb3d8bbwe\LocalCache\Roaming\Microsoft\`
 6. Delete the **Teams** folder.
-7. Restart the Teams Room device.
+7. Restart the Teams Rooms device.
 8. Sign in and restart again, once the Teams Room's interface has appeared.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-operations.md:262]:
```suggestion
8. Sign in and restart again, once the Teams Rooms interface has appeared.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the additional suggested change. Many thanks.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9  - Thanks for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**